src/obstacle_avoidance/include/MyNode.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: July 16th, 2016
+ * Description: An example node that subscribes to a topic publishing strings,
+ *              and re-publishes everything it receives to another topic with
+ *              a "!" at the end
+ */
+
+#ifndef SAMPLE_PACKAGE_MYNODE_H
+#define SAMPLE_PACKAGE_MYNODE_H
+
+#include "geometry_msgs/Twist.h"
+#include "sensor_msgs/LaserScan.h"
+#include <cmath>
+#include <iostream>
+#include <ros/ros.h>
+#include <sb_utils.h>
+#include <std_msgs/String.h>
+
+#define LIDAR_ARRAY_SIZE 100
+#define VEHICLE_WIDTH 0.35      // metres
+#define MIN_CRITICAL_DISTANCE 2 // metres
+#define ANG_VEL_GAIN 1.2
+#define WALL_HUG_VEL 0.8
+#define LINEAR_VEL 0.35
+#define MIN_GOAL_WIDTH 0.32
+#define MAX_GOAL_WIDTH 0.6
```

Comment:
All these parameters should be class members of the obstacle avoidance class, as that is the only place that they are used.